mmsplice/__init__.py

```diff
@@ -2,19 +2,14 @@
 
 """Top-level package for mmsplice."""
 
-__author__ = """Jun Cheng"""
+__author__ = """Jun Cheng & M.Hasan Celik"""
 __email__ = 'chengju@in.tum.de'
-__version__ = '0.2.7'
+__version__ = '0.2.8'
 
 from keras.models import load_model
-import mmsplice.generic
-import mmsplice.interval_tree
-import mmsplice.vcf_dataloader
-import mmsplice.exon_dataloader
-from .utils import postproc
 from mmsplice.mmsplice import MMSplice, \
-    predict_all, \
     writeVCF, \
+    predict_save, \
     predict_all_table, \
```

Comment:
I found the "assembly" argument is still useful.

Reply:
`predict_all_table` by default predict and return all submodule and assembled prediction so no need for `assembly` argument which can be confusing #13 . Also, assembly argument doesn't provide an additional benefit or significant performance improvement in current behavior.